types/ui_schema/field.ts

```diff
@@ -26,9 +27,19 @@ interface AssocitaionSettings {
   associationResource: string
 }
 
+interface FileSettings {
+  /** Size limit of single file with Byte unit, default Infinity */
+  maxFileItemSize?: number
```

Comment:
byte 感觉不是特别常用，填写的时候还要换算有点麻烦

```suggestion
  /** Defines the max size of a file upload in megabytes (MB) (inclusive) that can be selected for uploading, default Infinity */
  maxFileItemSizeLimitInMB?: number
```